llvm/lib/Target/RISCV/RISCVSubtarget.h

```diff
@@ -138,6 +155,8 @@ class RISCVSubtarget : public RISCVGenSubtargetInfo {
   /// initializeProperties().
   RISCVProcFamilyEnum getProcFamily() const { return RISCVProcFamily; }
 
+  RISCVProfileEnum getRISCVProfile() const { return RISCVProfile; }
+
```

Comment:
I guess it little bit same situation like those macro extensions, so what if user manually specify all extension present in certain profile? does it will detect right when invoke `getRISCVProfile`?

Reply:
Curently no, but we can support them in `RISCVISAInfo::updateCombination()`.
And I don't know if we need `getRISCVProfile` function, maybe we should remove it because there is no user.